react-function-components/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton(props) {
+  return <button>Click Me!</button>;
+}
+
+const element = <CustomButton/>;
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+root.render(element);
```

Comment:
You don't have to save the component into a variable, you can directly pass the component to `render`:

```JSX
root.render(<CustomButton />);
```